backend/src/http/handlers.rs

```diff
@@ -197,6 +181,27 @@ pub(super) async fn reply_404(ctx: &Context, method: &Method, path: &str) -> Res
     ctx.assets.serve_index(StatusCode::NOT_FOUND, &ctx.config).await
 }
 
+async fn handle_rss_request(path: &str, ctx: &Arc<Context>) -> Result<Response, Response> {
+    let Some(series_id) = path.strip_prefix("/~rss/series/") else {
+            return Ok(Response::builder()
+                .status(StatusCode::NOT_FOUND)
+                .body("Not found".into())
+                .unwrap());
+    };
+
+    let rss_content = match rss::generate_feed(&ctx, series_id).await {
+        Ok(rss_content) => rss_content,
+        Err(_) => {
+            return Err(response::internal_server_error());
+        }
+    };
+
+    Ok(Response::builder()
+        .header("Content-Type", "application/rss+xml")
+        .body(Body::from(rss_content))
+        .unwrap())
+}
```

Comment:
I'm not too sure that's what you meant in your [comment](https://github.com/elan-ev/tobira/pull/959#discussion_r1360812034) on this. Neither if this also needs to call the `error!` macro like it is done in the `handle_api` function.